### PR TITLE
Build structure without contacts

### PR DESCRIPTION
**Pull request by RajatMCAF**
This is fundamental model. Andri is still working on combining the funtionality.

### CODE REVIEW

File: crm_custom/views/views.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                        </page>
+                    </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="contact_creation_wizard_form_view" model="ir.ui.view">
+        <field name="name">client.creation.wizard.form</field>
+        <field name="model">contact.creation.wizard</field>
```
Comment (MORPSmcaf) [inline crm_custom/views/views.xml:28]: rename contact to client

File: crm_custom/views/views.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        <field name="name">Households</field>
+        <field name="res_model">household</field>
+        <field name="view_mode">tree,form</field>
+        <field name="help" type="html">
+            <p class="oe_view_nocontent_create">Click 'Create' to add a new household.</p>
+        </field>
+    </record>
+
+
+    <menuitem id="menu_root_crm" name="Custom CRM" sequence="1"/>
```
Comment (MORPSmcaf) [inline crm_custom/views/views.xml:55]: move to 'views/menu.xml'

File: crm_custom/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```
[the remark this answers is not in the document]
Comment (RajatMCAF) [reply crm_custom/__manifest__.py:1]: Done

File: crm_custom/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+{
+    'name': "Crm_custom",
```
[the remark this answers is not in the document]
Comment (RajatMCAF) [reply crm_custom/__manifest__.py:3]: Done

File: crm_custom/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    'summary': "Tamplate",
```
[the remark this answers is not in the document]
Comment (RajatMCAF) [reply crm_custom/__manifest__.py:4]: Done

File: crm_custom/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    'author': "MCAF",
+    'website': "https://mcaf.nb.ca/en/",
+    'author': "MCAF",
+    'category': "CRM Internal",
+    'license': 'OPL-1',
+    "application": True,
+    "installable": True,
+    'version': '0.5',
```
[the remark this answers is not in the document]
Comment (RajatMCAF) [reply crm_custom/__manifest__.py:12]: Done

File: crm_custom/__manifest__.py [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+    'depends': ['base', 'mail'],
+
+    'data': [
+        'security/ir.model.access.csv',
+        'views/views.xml',
```
[the remark this answers is not in the document]
Comment (RajatMCAF) [reply crm_custom/__manifest__.py:17]: Done

File: crm_custom/models/list_of_users.py [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+    partner_name = fields.Char('Name', required=True)
+    phone = fields.Char('Phone')
+    email = fields.Char('Email')
+    household_id = fields.Many2one('household', string="Household", readonly=True,
+                                   default=lambda self: self._default_household())
+
+    def _default_household(self):
+        return self.env.context.get('default_household_id')
+
+    def create_contact(self):
```
[the remark this answers is not in the document]
Comment (RajatMCAF) [reply crm_custom/models/list_of_users.py:42]: Done

File: crm_custom/security/ir.model.access.csv [hunk 9]
```diff
@@ -0,0 +1,5 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+access_household,access_household,model_household,base.group_user,1,1,1,1
+access_contact_creation_wizard,access_contact_creation_wizard,model_contact_creation_wizard,base.group_user,1,1,1,1
```
[the remark this answers is not in the document]
Comment (RajatMCAF) [reply crm_custom/security/ir.model.access.csv:3]: Done

File: crm_custom/views/views.xml [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+                                </tree>
+                            </field>
+                        </page>
+                    </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="contact_creation_wizard_form_view" model="ir.ui.view">
```
[the remark this answers is not in the document]
Comment (RajatMCAF) [reply crm_custom/views/views.xml:26]: Done

File: crm_custom/views/views.xml (hunk 1 above)
Comment (RajatMCAF) [reply crm_custom/views/views.xml:28]: Done

File: crm_custom/views/views.xml (hunk 2 above)
Comment (RajatMCAF) [reply crm_custom/views/views.xml:55]: Done

**Review by MORPSmcaf — APPROVED** [review]

**Pull request merged**